src/client/apps/edit/components/edit_container.jsx

```diff
@@ -205,7 +206,7 @@ const mapDispatchToProps = {
   toggleSpinnerAction: toggleSpinner
 }
 
-export default connect(
+export default hot(module)(connect(
```

Comment:
Ooo so we have to do this each time now? Seems like it could get a little tedious to remember - like i will probably just forget to use HMR entirely 😛. It's ok tho, we can move forward with this and come up with something smoother in the future.

Reply:
@kanaabe @eessex  - it only needs to be added into root components -- so right now its `/list` and `/edit` -- from there it spreads down the component tree. In prod, `hot` is just a noop pass-through method so no need to worry about removing.